packages/composites/dydx-rewards/schemas/env.json

```diff
@@ -12,6 +12,12 @@
       "type": "string",
       "format": "uri"
     },
+    "ETHEREUM_CHAIN_ID": {
+      "required": false,
```

Comment:
Should this be required?

Reply:
Since there is a default value if not provided, we don't really need this variable to be required. Assuming most of the times connection is made to mainnet. Also for other adapters as well

Reply:
Let's make it not required as long as there's a default set.